_docs/usecases/kubernetes/backups.md

```diff
@@ -0,0 +1,143 @@
+
+This example is provided to give ideas of how to exfiltrate files that have
+been written to a StorageOS persistent volume. This example uses nginx, rsync
+and sftp sidecars in an application pod. This approach may be useful to create
+out-of-cluster backups of data.
+
+In this example the "application" container is the container "main", which has
+rsync, nginx and sftp sidecar containers. The StorageOS volume that the
+application is writing to will be mounted into the sidecar containers as well so
+backup files etc. can be exfiltrated. 
+
+The files create a stateful set that can be used *AFTER* a StorageOS cluster
+has been created. For more information on how to install a StorageOS cluster
+please see [the StorageOS documentation]
+(https://docs.storageos.com/docs/introduction/quickstart).
+
+## Deploy
+
+In order to deploy the pod clone this repostiory and use kubectl to create the
+Kubernetes objects.
+
+> Before deploying the backup-example stateful set we highly recomend looking
+> through the examples to understand how the different containers are
+> configured 
+
+```bash
+$ git clone https://github.com/storageos/deploy.git storageos
+$ cd storageos
+$ kubectl create -f ./k8s/examples/backup-example
+```
+Check that a backup-example pod is running
+
+```bash
+$ kubectl get pods -w -l app=backup-example
+   NAME        READY    STATUS    RESTARTS    AGE
+   backup-example-0     1/1      Running    0          1m
+```
+
+## Exfiltrating files through Nginx
+
+Connect to the Nginx container running inside the backup-example pod through
+the service that's configured in 10-service.yaml. In this example we do this by
+executing wget from inside a Busybox pod.
+
+```bash
+$ kubectl exec -it busybox -- /bin/sh
+/ # /bin/busybox wget backup-example
+Connecting to backup-example (100.64.159.6:80)
+index.html           100% |********************************|   367  0:00:00 ETA
+/ # cat index.html
+<html>
+<head><title>Index of /</title></head>
+<body>
+<h1>Index of /</h1><hr><pre><a href="../">../</a>
+<a href="helloworld.txt">helloworld.txt</a>  06-Nov-2018 14:42      12
+<a href="date.txt">test.txt</a>              06-Nov-2018 14:54      16
+</pre><hr></body>
+</html>
+/ # /bin/busybox wget backup-example/date.txt
+Connecting to backup-example (100.64.159.6:80)
+date.txt             100% |********************************|    56  0:00:00 ETA
+```
+
+Depending on what files have been written to the StorageOS volume the output of
+the index file will be different. In the example above two .txt files were
+present on the volume.
+
+## Exfiltrating files through Rsync
+
+Connect to the Rsync daemon, that's running inside the backup-example pod,
+through the service. In the example below we do this by conencting to an rsync
+pod. A username and password that are set in the rsync-credentials secret. The
+secret supplied in the example has the username and password set to username
+and password.
+
+```
+$ keti rsync -- sh
```

Comment:
s/keti/kubectl exec -it rsync -- sh/

Even readers know what keti alias means, it is not elegant.